pkg/pillar/cmd/zedrouter/ipaddronunet.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Network Instance underlay network Application Number Management
+
+package zedrouter
+
+import (
+	"fmt"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+)
+
+func appNumsOnUNetAllocate(ctx *zedrouterContext,
+	config *types.AppNetworkConfig) {
+	isStatic := false
```

Comment:
This is bad with the clear at the end of the loop.

Just set it with isStatic := (ulConfig.AppIPAddr != nil) inside the loop